### PR TITLE
Password change

### PR DESCRIPTION
**Pull request by SMCIND**

### CODE REVIEW

File: src/app/service/http-client.service.ts [hunk 1]
```diff
@@ -123,10 +123,25 @@ errorHandl(error) {
...
     retry(1),
-    catchError(this.errorHandl)
+    catchError(this.errorHandl),
+    
   )
 }  
+// PUT
+ChangeBusinessPassword(data): Observable<Business> {
+  //console.log(123,this.currentBusiness);
+  console.log(data);
```
Comment (saigoud1000) [inline src/app/service/http-client.service.ts]: Why are logging here?

File: src/app/find-business/find-business.component.html [hunk 2]
```diff
@@ -57,7 +57,7 @@ <h4 class="card-title font-weight-bold">
                     </h4>
                     
                     <h6>
-                      <div class="rating mt-2">
+                      <div class="rating mt-2" title="Sign Up as Customer to add rating for the business">
```
Comment (saigoud1000) [inline src/app/find-business/find-business.component.html]: This is not working

File: src/app/service/http-client.service.ts (hunk 1 above)
Comment (SMCIND) [reply src/app/service/http-client.service.ts]: its just a check, whether request is properly framed in front-end or not. Need to be removed!

File: src/app/find-business/find-business.component.html (hunk 2 above)
Comment (SMCIND) [reply src/app/find-business/find-business.component.html]: Will check this but it is working for me.

**Pull request merged**